src/main/java/frc/robot/commands/ArmCommands/AimArm.java

```diff
@@ -0,0 +1,40 @@
+package frc.robot.commands.ArmCommands;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.subsystems.ArmSubsystem;
+import frc.robot.subsystems.PeterSubsystem;
+
+public class AimArm extends Command {
+  private ArmSubsystem armSubsystem;
+
+  public AimArm(ArmSubsystem armSubsystem) {
+    this.armSubsystem = armSubsystem;
+    addRequirements(armSubsystem);
+  }
+
+  // Called when the command is initially scheduled.
+  @Override
+  public void initialize() {}
+
+  // Called every time the scheduler runs while the command is scheduled.
+  @Override
+  public void execute() {
+    if (PeterSubsystem.noteStaticPresent()) {
```

Comment:
why note static present? take in peter, don't add as requirement, and use notePresent();